bpm/bonita-core/bonita-process-engine/src/main/java/org/bonitasoft/engine/bpm/model/impl/BPMInstancesCreator.java

```diff
@@ -546,72 +545,66 @@ public void createConnectorInstances(final PersistentObject container, final Lis
         final List<SConnectorInstance> connectorInstances = new ArrayList<SConnectorInstance>(connectors.size());
         int executionOrder = 0;
         for (final SConnectorDefinition sConnectorDefinition : connectors) {
-            connectorInstances.add(BuilderFactory
-                    .get(SConnectorInstanceBuilderFactory.class)
-                    .createNewInstance(sConnectorDefinition.getName(), container.getId(), containerType,
-                            sConnectorDefinition.getConnectorId(), sConnectorDefinition.getVersion(), sConnectorDefinition.getActivationEvent(),
-                            executionOrder++)
-                    .done());
+            final int executionOrder1 = executionOrder++;
```

Comment:
`executionOrder++` increments after use, which changes the previous behaviour. Are u sure this is what you want?